src/server/components/lists/controllers/ingest/ingestPutController.ts

```diff
@@ -48,15 +52,15 @@ export async function ingestPutController(req: Request, res: Response) {
       },
     });
 
-    if (listItem === null) {
-      return res.status(404).send({
+    if (listItem === undefined) {
+      res.status(404).send({
```

Comment:
does the form runner actually do anything with this error if it is produced?

Reply:
although thanks for the spot. these should all be `return res...`